.circleci/config.yml

```diff
@@ -626,3 +658,7 @@ workflows:
         name: test << matrix.nginx-version >> on << matrix.base-image >>:<< matrix.arch >> WAF << matrix.waf >>
         requires:
         - build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
+    - system_tests:
+        name: Run system tests
+        requires:
+        - build 1.25.4 on amd64 WAF ON
```

Comment:
This one is not necessary.

Reply:
🤔 it's necessay for `build-and-test` but not for `build-and-test-all` ?